pages/4_Custom_Case.py

```diff
@@ -0,0 +1,40 @@
+# pylint: disable=invalid-name
```

Comment:
I think this name could be a bit clearer - what do you think about `Experiment with Changes`?